### ISSUE TITLE
Not encouraging the use of double-sided ruby

### ISSUE THREAD

**Issue by murata2makoto** [w3c/ruby-tts-req#57]
No browsers support double-sided ruby.  Moreover, JLTF does not plan to cover the details of it in the upcoming JLReq-D.

How about dropping all subclasses about double-sided ruby?  We might want to keep 2.6 without providing details of double-sided ruby.

**Comment by r12a** [comment]
The HTML WG is about to publish the spec that will eventually be integrated into HTML5.  A result of that should be that browsers do implement support for double-sided ruby.

**Comment by frivoal** [comment]
> No browsers support double-sided ruby.

Not true. Firefox does using `<rtc>`, and all browsers do [using nested ruby](http://software.hixie.ch/utilities/js/live-dom-viewer/?%3C!DOCTYPE%20html%3E%0A%3Cstyle%3E%0A%20%20body%3Eruby%20%7B%0A%20%20ruby-position%3A%20under%3B%0A%7D%0Aruby%3Eruby%20%7B%0A%20%20ruby-position%3A%20over%3B%0A%7D%0A%20%20%3C%2Fstyle%3E%0A%3Cbody%3E%0A%20%20%3Cruby%3E%3Cruby%3EA%3Crt%3Eb%3C%2Fruby%3E%3Crt%3Ec%3C%2Fruby%3E%0A). Nested ruby is less expressive than `<rtc>`, and has a number of issues, but regardless, double sided ruby is possible in all browsers, so looking into the limitations of current solutions seems relevant.

How much emphasis should be on it, how much priority it should be given is a different question.

**Comment by murata2makoto** [comment]
> Not true. Firefox does using <rtc>

Stand corrected. 

No existing subclasses in this draft about double-sided ruby consider `<rtc> `or nested ruby.   I am going to drop these subclasses now and think about reintroducing them when double-sided ruby is introduced to W3C and WHATWG HTML.

**Comment by murata2makoto** [comment]
Section [2.6](https://w3c.github.io/ruby-t2s-req/#double-sided-ruby-background) explains the concept of double-sided ruby.  Each of the three approaches has its own advantages and disadvantages when applied to double-sided ruby.  However, these pros and cons cannot be fully understood without first clarifying the underlying markup structure.  In fact,  Sections [3.1.6](https://w3c.github.io/ruby-t2s-req/#double-sided-ruby-when-both-read-aloud), [3.2.6](https://w3c.github.io/ruby-t2s-req/#double-sided-ruby-when-ruby-annotations-read-aloud), and [3.3.6](https://w3c.github.io/ruby-t2s-req/#double-sided-ruby-when-bases-read-aloud) provide only a preliminary discussion of this issue.  

When I wrote the above comment, I initially thought these clauses should simply be deleted.  However, I now believe it is more helpful to retain them and add the same editorial note to Sections [3.1.6](https://w3c.github.io/ruby-t2s-req/#double-sided-ruby-when-both-read-aloud), [3.2.6](https://w3c.github.io/ruby-t2s-req/#double-sided-ruby-when-ruby-annotations-read-aloud), and [3.3.6](https://w3c.github.io/ruby-t2s-req/#double-sided-ruby-when-bases-read-aloud) to indicate that the descriptions are provisional and require further refinement.

**Note:** Further discussion is not possible until the markup representation of double-sided ruby is clearly defined.
